controllers/extendeddaemonset_test.go

```diff
@@ -28,9 +28,9 @@ import (
 	// +kubebuilder:scaffold:imports
 )
 
-// This test may take ~30s to run, check you go test timeout
+// This test may take ~60s to run, check you go test timeout
 var _ = Describe("ExtendedDaemonSet Controller", func() {
-	const timeout = time.Second * 30
+	const timeout = time.Second * 60
```

Comment:
The 30s timeout is per "step" (usually for each `Eventually` part), there might be an issue if it takes up to 60s for a single part

Reply:
30s seems to be enough locally and in gitlab, in github it timed out once though that's why I doubled the timeout

Reply:
I've put it back to 30s and it seems to be working fine 🤷‍♂️